Objects/Converters/ConverterCSI/ConverterCSIShared/PartialClasses/Properties/Convert2DPropertyWall.cs

```diff
@@ -9,7 +10,7 @@ public partial class ConverterCSI
   {
     public string WallPropertyToNative(CSIProperty2D Wall)
     {
-      throw new Exception("Wall properties are not currently supported on receive");
+      throw new ConversionSkippedException("Wall properties are not currently supported on receive");
```

Comment:
Good point, I think I immediately regretted naming the exception as such, as it describes what WILL happen, not what HAS happened, which is a no no for exceptions (exceptions shouldn't care how they are handled)

In Revit connector, we were abusing this exception type a little, to correct for ambiguity in how objects should be traversed. (atleast, imo)

Here, I feel much more ok, since we are quite simply saying "This thing, we don't support it" rather than the revit cases which were "This thing is supported, but were skipping it in some places because we can't skip it anywhere else"

For this reason, I think creating a new exception `ConversionNotSupportedException` or `UnsupportedConversionException` (do you have a preference?) for this case. What do you think?

Reply:
@BovineOx @AlanRynne @clairekuang what do you think to a new exception along these lines (for 2.17). This we use in Rhino & CSI (and everywhere else). Then in Revit, we keep the current usage of `ConversionSkippedException` (since changing it doesn't quite fit with how we're using it)

Reply:
I'd approve of a `ConversionNotSupported` exception, and your suggestion of moving on with that except for Revit (which we'd do at some point in the future...)

Reply:
Ok, I've added `ConversionNotSupportdException`
And replaced the usages in the CSI codebase only (left Revit untouched)